rules/linux-kernel.mk

```diff
@@ -7,8 +7,19 @@ KERNEL_SUBVERSION = 1+deb9u5
 
 export KVERSION_SHORT KVERSION KERNEL_VERSION KERNEL_SUBVERSION
 
+
+SMPATH=$(SRC_PATH)/sonic-linux-kernel/
+SM_DEP_LIST  := Makefile
+SM_DEP_LIST  += patch/*
+SM_DEP_LIST  += patch/preconfig/*
```

Comment:
why do we have only rule for linux-kernel? We should have rules for other source code?